### DISCUSSION TITLE
How to disable left-side tabs menu on mobile

### DISCUSSION THREAD

**Discussion by msoutopico** [squidfunk/mkdocs-material#4877] [Q&A] [answered]
Hi there, 

I'm using the `navigation.tabs` feature where each tab is the header to one of the main pages. However, I actually hide the tabs bar using CSS because I only want each user to see their page and not be able to open pages for other users.

That works file, but when the page is shrunk on a mobile device, the tabs appear on the left-hand side menu when the user clicks on the three-line burger icon. 

How can I disable that behaviour on mobile? 
Thanks in advance.

PS: I have tried hiding the burger icon (`.md-icon`) but that seems to truncate the name of sections in the next/prev footer.

**Comment by squidfunk** [comment, marked as answer]
I'm not sure if Material for MkDocs is right for your use case, because it doesn't sound like something we're optimizing for, but you should be able to hide the button with:

``` css
.md-header__button[for="__drawer"] {
  display: none
}
```

**Reply by msoutopico** [reply]
Thank you so much, Martin! That seems to do the trick just fine. 

Yes, a lot of specificities in my use case, but Material for MkDocs really helps in many other regards with my requirements. 

Cheers!
